### PR TITLE
Fix migration upgrade handler

### PR DESCRIPTION
**Pull request by amaurym**

### CODE REVIEW

File: app/experimental_appconfig.go [hunk 1]
```diff
@@ -73,13 +73,45 @@ func setCustomModules(app *RegenApp, interfaceRegistry types.InterfaceRegistry)
...
+		panic(err)
+	}
+
+	if upgradeInfo.Name == "v0.43.0-beta1-upgrade" && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
+		storeUpgrades := storetypes.StoreUpgrades{
+			Added: []string{"authz", "feegrant"},
+		}
+
+		// configure store loader that checks if version == upgradeHeight and applies store upgrades
+		app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
```
Comment (amaurym) [inline app/experimental_appconfig.go:113]: Should we add this logic into the SDK directly? In `RunMigrations` more specifically, if `fromVM["modulename"]`==0, then we add the storeloader, and we run InitGenesis

File: app/experimental_appconfig.go [hunk 2]
```diff
@@ -73,13 +73,45 @@ func setCustomModules(app *RegenApp, interfaceRegistry types.InterfaceRegistry)
 }
 
 func (app *RegenApp) registerUpgradeHandlers() {
-	app.UpgradeKeeper.SetUpgradeHandler("v0.43.0-beta1-upgrade", func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
-
-		// skipping x/auth migrations. It is already patched in regen-ledger v1.0
-		fromVM["auth"] = auth.AppModule{}.ConsensusVersion()
+	app.UpgradeKeeper.SetUpgradeHandler("v0.43.0-beta1-upgrade", func(ctx sdk.Context, plan upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
+		// 1st-time running in-store migrations, using 1 as fromVersion to
```
Comment (amaurym) [inline app/experimental_appconfig.go:77]: btw, shouldn't all this be copied to stable_appconfig.go too? As I see,
- stable: v043, authz, feegrant
- experimental: above + group, data, ecocredit

Comment (anilCSE) [reply app/experimental_appconfig.go:77]: Ah okay. I thought we are still keeping authz in expermental. Yes, let's move them to stable config

Comment (anilCSE) [reply app/experimental_appconfig.go:77]: we should also create an issue to remove this upgrade handler before tagging rc-1

File: app/experimental_appconfig.go (hunk 1 above)
Comment (anilCSE) [reply app/experimental_appconfig.go:113]: That's better UX. I would advise to do that. But how are we gonna pass the storekeys there?

Comment (amaurym) [reply app/experimental_appconfig.go:113]: Let's discuss here: https://github.com/cosmos/cosmos-sdk/issues/9381

**Review by aleem1314 — APPROVED** [review]

**Review by anilCSE — APPROVED** [review]
lgtm

**Pull request merged**